packages/classic-webview/src/components/UnlockHardcoreCTAContent.tsx

```diff
@@ -18,7 +18,7 @@ export const UnlockHardcoreCTAContent = ({
         Unlocking Hardcore grants access to today and all previous hardcore puzzles.
```

Comment:
Can you make the text centered?
<img width="415" alt="image" src="https://github.com/user-attachments/assets/9fc58d5f-a1c5-448a-84ca-bcdedf499412" />